qa/python_models/custom_metrics/model.py

```diff
@@ -136,6 +226,27 @@ def test_gauge_e2e(self):
         metrics = self._get_metrics()
         self.assertIn(pattern, metrics)
 
+    def test_histogram_e2e(self):
```

Comment:
Would be good to have some negative tests with no buckets provided to ensure no crash/hang:
```python
metric = metric_family.Metric(labels=labels)
metric = metric_family.Metric(labels=labels, buckets=[])
metric = metric_family.Metric(labels=labels, buckets=None)
```

Reply:
Added all tests